_docs/style/text-conventions/general-guidelines.md

```diff
@@ -0,0 +1,194 @@
+| We offer you a comprehensive portfolio of hosting options. | Rackspace offers a comprehensive portfolio of hosting options for the enterprise buyer. |
+| Cloud Backup uses block-level deduplication, which means that only those parts of a file that have changed are saved. | Cloud Backup uses block-level deduplication, which means we save only those parts of a file that have changed. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Avoid abbreviations
+
+Avoid abbreviations, acronyms, and slang. Use an abbreviation only if it is better known than the word that it stands for or if space savings is critical.
+
+Unless an abbreviation is common, spell out the words of the abbreviation on the first use. Show the abbreviation in parentheses after the spelled-out term; for example, access control list (ACL). On subsequent uses, use the abbreviation. If you introduce an abbreviation, use it; don't alternate between the abbreviation and the spelled-out term.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Maintain parallel construction
+
+Parallel construction means that each part of a sentence uses the same grammatical structure. When writing a series, it is important that each element in the series performs the same function, that of the object of the verb. For example:
+
+{% figure [caption:"Provide context example"] [class:"image bg-light border"] %}
+| Use | Avoid  |
+| --- | --- |
+| This stack includes a Cloud Load Balancer, a Cloud Database, and a Master server (plus optional secondary servers). | This stack includes a Cloud Load Balancer, Cloud Database, and a Master server (plus optional secondary servers). |
+{: class="hxTable"}
+{% endfigure %}
+
+In the correct example, notice the **a** before Cloud Database.
+
+Parallel structure also applies to options you present to users. Don't ask users to choose from unrelated values. For example, when providing radio button options:
+
+ {% figure [caption:"Provide context example"] [class:"image bg-light border"] %}
+ | Use | Avoid  |
+ | --- | --- |
+ | **Always**, **Only with streaming**, and **Never**. | **Always**, **Only with streaming**, and **Blocked**. |
+ {: class="hxTable"}
+ {% endfigure %}
+
+In this example, **Always** and **Never** are opposites. **Always** and **Blocked** are not opposites.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Provide context
```

Comment:
the caption on the example image is confusing since the section is called "Provide context" and the caption is merely identifying the section rather than telling the reader to "provide ____" as a declarative verb.... it just reads a little weird

Reply:
That's mostly how I've used captions, to identify the section and then add the word 'examples'. it's not really mean to be declarative, I'm just identifying the content that's contained in the table. So 'Provide context examples' is consistent with that approach. We can reconsider how we're using captions, or remove them entirely, since I don't feel like they add a lot of value. Though, removing them would be inconsistent with using captions throughout the entire site. Let's chat more about this...